lib/config/base.js

```diff
@@ -69,14 +69,9 @@ module.exports = {
         // Throw errors for unnecessary usage of .call or .apply
         'no-useless-call': 'error',
 
-        // Arrow functions
-        // Have parens for multiple args and start on the same line
-        // .do((something, else) => {
-        //    ...many lines...
-        //  });
-        // Unless it's a very simple one-line rule
-        // .catch(err => reject(err))
-        'arrow-body-style': ['error'],
+        // Do not enforce single lines when using arrow functions.
+        // https://eslint.org/docs/rules/arrow-body-style
+        'arrow-body-style': 0,
```

Comment:
Should this be `'arrow-body-style': 'off',` to match the format of the rest of the rules?